README.md

```diff
@@ -1 +1,45 @@
 # hello_microverse
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Project to demonstrate mastery of coding and collaboration best practices.
+
+
+## Built With
+
+- HTML and CSS
+
+
+## Getting Started
+
+
+To get a local copy up and running just download and open on a browser.
+
+
+
+## Author
+
+- GitHub: [@githubhandle](https://github.com/emmiiorji)
+- Twitter: [@twitterhandle](https://twitter.com/emmiiorji)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/emmanuel-orji-2a8317121)
```

Comment:
- [ ] [OPTIONAL] Kindly Add your name under the "Author" title and change each handler to your handler's name for that social platform.

As an example, if you'd written:
```
GitHub: [@emmiiorji](https://github.com/emmiiorji)
```
it would appear as such: 
GitHub: [@emmiiorji](https://github.com/emmiiorji)